modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * updating/index.adoc
+
+:_content-type: CONCEPT
+[id="Understanding_clusteroperator_conditiontypes_{context}"]
+= Understanding cluster operator condition types
+
+The status of cluster operators is revealed with the condition type, informing you of potential impacts to upgrading your cluster. The following definitions cover a list of some common ClusterOperator condition types. Some operators may have additional condition types with operator-specific language, which are not included here. 
```

Comment:
> informing you of potential impacts to upgrading your cluster

status conditions are just not about upgrades. An operator can degraded any time during the life of a cluster. Just not during upgrades. So we should say `informing you of current state of operator health`